util.py

```diff
@@ -51,43 +50,96 @@ def find_closest_servers(self, location, servers_to_search = None):
                 raise ValueError('Server <' + server + '> latitude/longitude could not be found!')
             server_location = geopy.Point(server_lat_lon[0], server_lat_lon[1])
             server_dict['distance'] = great_circle(item_location, server_location).km
-
             best_servers.append(server_dict)
-
         best_servers.sort(key=self.get_distance_key)
-
     return best_servers
 
+# Converts the local hostname to the simulation ip address
+#
+# params:
+#   local_host: the local hostname
+def convert_to_simulation_ip(local_host):
+    if !os.path.exists(SIMULATION_IP_FILE):
+        return None
+    result = None
+    with open(SERVER_LIST_FILE, 'rb') as server_file, open(SIMULATION_IP_FILE, 'rb') as simulation_ip_file:
+        for line in server_file:
+            simulation_ip = simulation_ip_file.next()
+            if line == localhost:
+                result = simulation_ip
+                break
+    return result
+
+# Converts the simulation ip address to the local hostname
+#
+# params:
+#   local_host: the local hostname
+def convert_to_local_hostname(simulation_ip):
+    if !os.path.exists(SIMULATION_IP_FILE):
+        return None
+    result = None
+    with open(SERVER_LIST_FILE, 'rb') as server_file, open(SIMULATION_IP_FILE, 'rb') as simulation_ip_file:
+        for line in simulation_ip_file:
+            local_hostname = server_file.next()
+            if line == simulation_ip:
+                result = local_hostname
+                break
+    return result
+
+def convert_localhost_to_simulation_ip(server):
+    # hardcode AWS servers for simulation
+    if server == 'localhost:5000':
+      server = '54.175.68.60'
+    elif server == 'localhost:5001':
+        server = '54.65.80.55'
+    elif server == 'localhost:5002':
+        server = '54.93.104.58'
+    elif server == 'localhost:5003':
+        server = '54.207.24.208'
+    elif server == 'localhost:5004':
+        server = '54.69.237.99'
+    return server
+
+def convert_server_to_test_server(server):
```

Comment:
Ahh.. sure. I wasn't sure where this method is being called and I don't want to break things so I was thinking that may be we can have the `convert_to_local_hostname` in master and have each one of use change to that independently.